plugins/janus_videoroom.c

```diff
@@ -376,16 +377,29 @@ room-<unique room ID>: {
 			"room" : <unique numeric ID>,
 			"description" : "<Name of the room>",
 			"pin_required" : <true|false, whether a PIN is required to join this room>,
+			"is_private" : <true|false, whether this room is 'private' (as in hidden) or not>,
 			"max_publishers" : <how many publishers can actually publish via WebRTC at the same time>,
 			"bitrate" : <bitrate cap that should be forced (via REMB) on all publishers by default>,
-			"bitrate_cap" : <true|false, whether the above cap should act as a limit to dynamic bitrate changes by publishers>,
+			"bitrate_cap" : <true|false, whether the above cap should act as a limit to dynamic bitrate changes by publishers (optional)>,
```

Comment:
Not sure why you added the optional there: basically all the other propeerties are as well. Please remove this.

Reply:
I see some of the other properties have it too, possibly because of copy/pasting from the config documentation. I think they should all be removed, since they incorrectly suggest other properties are mandatory instead, many of which aren't.

Reply:
I can remove the "optional" part if you want.

That said, if you compare to the actual code here [here](https://github.com/meetecho/janus-gateway/blob/36b85ee61806c92caa267d39a64094375007b4c7/plugins/janus_videoroom.c#L3764), you will notice that all the fields which I have not marked with `(optional)` are always set, so the 

> since they incorrectly suggest other properties are mandatory instead, many of which aren't.

part is not true, unless there is some logic somewhere which removes empty fields in returned JSON which I did not catch.

Reply:
Fair point, I got confused with `create` :slightly_smiling_face: